### PR TITLE
Cleaning up in test_series.py

### PR DESCRIPTION
**Pull request by kozlov-alexey**

### CODE REVIEW

File: sdc/tests/test_series.py [hunk 1]
```diff
@@ -1322,7 +1272,26 @@ def test_impl(A, n):
                     S = pd.Series([2, 4, 6, 6, 3], index)
                     pd.testing.assert_series_equal(hpat_func(S, n), test_impl(S, n))
 
-    @unittest.expectedFailure
+    '''
```
Comment (densmirn) [inline sdc/tests/test_series.py]: [PEP8](https://www.python.org/dev/peps/pep-0008/#string-quotes): For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.

Please fix it here and in some places below.

Comment (kozlov-alexey) [reply sdc/tests/test_series.py]: Sure, changed in all places.

**Comment by AlexanderKalistratov** [comment]
@kozlov-alexey travis has failed

**Comment by AlexanderKalistratov** [comment]
please add name to all getitem tests. As far as I see not all cases are covered with names

**Comment by kozlov-alexey** [comment]
@AlexanderKalistratov 
I fixed travis and added names in tests for getitem/loc/iloc. Can be merged once CI passes.

**Comment by AlexanderKalistratov** [comment]
@kozlov-alexey it seems that travis still failing?

**Comment by kozlov-alexey** [comment]
@AlexanderKalistratov Yes, it seems I've been fooled by our tests (they passed on my local machine in old pipeline run). Now I see that for the rest 3 failing tests the reason is the same as for the others I skipped:
"During: resolving callee type: type(CPUDispatcher(<function get_series_data at 0x7f91f9506200>))"
Which is probably related to migration to latest Numba, so I have just skipped them.

**Comment by AlexanderKalistratov** [comment]
@kozlov-alexey conflict

**Comment by AlexanderKalistratov** [comment]
@kozlov-alexey cumulative sum for strings failed

**Pull request merged**